torch_geometric/nn/aggr/base.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+from typing import Optional
+
+import torch
+from torch import Tensor
+from torch_scatter import scatter, segment_csr
+
+
+class BaseAggr(torch.nn.Module, ABC):
```

Comment:
I would also just call this `Aggr` instead of `BaseAggr`.

Even further, making the class name descriptive as `Aggregation` or `Aggregator` would make it way easier to read. Not like we have a limit on screen space :)